package.json

```diff
@@ -274,7 +274,7 @@
     "@metamask/bridge-controller": "25.1.0",
     "@metamask/bridge-status-controller": "patch:@metamask/bridge-status-controller@npm%3A21.0.0#~/.yarn/patches/@metamask-bridge-status-controller-npm-21.0.0-7178a3f32f.patch",
     "@metamask/browser-passworder": "^4.3.0",
-    "@metamask/chain-agnostic-permission": "^0.6.0",
+    "@metamask/chain-agnostic-permission": "^0.8.0",
```

Comment:
Looks like the problem is that the version of `chain-agnostic-permission` we're using here is different from the version under test in the middleware pipeline.  Specifically the list of known EIp155 methods has changed. That list comes from `api-specs`. We're using `api-specs` v0.13.0 for the E2E tests, but the application is using v0.14.0.

We should be able to fix this by bumping `@metamask/api-specs` to v0.14.0

Reply:
Done here: https://github.com/MetaMask/metamask-extension/pull/33779/commits/eff7290eef14742dd4c259c0de97f6f3957457e0

Reply:
Looks like I was wrong, this wasn't the mismatch that was causing the problem here. It was the versions of `api-specs` between `metamask/chain-agnostic-permission` and `@metamask/multichain-api-middleware` that is the issue.

Given that we've discovered elsewhere that bumping `api-specs` to 0.14.0 causes test failures, perhaps we can undo the bump here, and instead bump `@metamask/multichain-api-middleware` to 0.3.0 here in this cherry-pick PR. And no equivalent change on `main` is needed, because it was bumped past there to 0.4.0 already on `main`: https://github.com/MetaMask/metamask-extension/pull/33511/

Reply:
Thanks for digging into this @Gudahtt . Yes that resolution makes more sense for this hotfix. I've bumped the `@metamask/multichain-api-middleware` version 0.3.0 here